featuretools/entityset/entityset.py

```diff
@@ -1121,52 +1121,61 @@ def add_interesting_values(self, max_values=5, verbose=False, dataframe_name=Non
 
         self.reset_data_description()
 
-    # def plot(self, to_file=None):
-    #     """
-    #     Create a UML diagram-ish graph of the EntitySet.
+    def plot(self, to_file=None):
+        """
+        Create a UML diagram-ish graph of the EntitySet.
 
-    #     Args:
-    #         to_file (str, optional) : Path to where the plot should be saved.
-    #             If set to None (as by default), the plot will not be saved.
+        Args:
+            to_file (str, optional) : Path to where the plot should be saved.
+                If set to None (as by default), the plot will not be saved.
 
-    #     Returns:
-    #         graphviz.Digraph : Graph object that can directly be displayed in
-    #             Jupyter notebooks.
+        Returns:
+            graphviz.Digraph : Graph object that can directly be displayed in
+                Jupyter notebooks.
 
-    #     """
-    #     graphviz = check_graphviz()
-    #     format_ = get_graphviz_format(graphviz=graphviz,
-    #                                   to_file=to_file)
+        """
+        graphviz = check_graphviz()
+        format_ = get_graphviz_format(graphviz=graphviz,
+                                      to_file=to_file)
 
-    #     # Initialize a new directed graph
-    #     graph = graphviz.Digraph(self.id, format=format_,
-    #                              graph_attr={'splines': 'ortho'})
+        # Initialize a new directed graph
+        graph = graphviz.Digraph(self.id, format=format_,
+                                 graph_attr={'splines': 'ortho'})
 
-    #     # Draw entities
-    #     for entity in self.dataframes:
-    #         variables_string = '\l'.join([var.id + ' : ' + var.type_string  # noqa: W605
-    #                                       for var in entity.variables])
-    #         if isinstance(entity.df, dd.DataFrame):  # entity is a dask entity
-    #             label = '{%s |%s\l}' % (entity.id, variables_string)  # noqa: W605
-    #         else:
-    #             nrows = entity.shape[0]
-    #             label = '{%s (%d row%s)|%s\l}' % (entity.id, nrows, 's' * (nrows > 1), variables_string)  # noqa: W605
-    #         graph.node(entity.id, shape='record', label=label)
-
-    #     # Draw relationships
-    #     for rel in self.relationships:
-    #         # Display the key only once if is the same for both related entities
-    #         if rel._parent_column_name == rel._child_column_name:
-    #             label = rel._parent_column_name
-    #         else:
-    #             label = '%s -> %s' % (rel._parent_column_name,
-    #                                   rel._child_column_name)
+        # Draw entities
+        for df in self.dataframes:
+            column_typing_info = []
+            for col_name, col_schema in df.ww.columns.items():
+                col_string = col_name + ' : ' + str(col_schema.logical_type)
+
+                tags = col_schema.semantic_tags - col_schema.logical_type.standard_tags
```

Comment:
In the context of a plot that will be easier to read if there's less to pick through for each column, the standard tags aren't usually adding much that the Logical Type doesn't already indicate (plus Variable types had no way to show this kind of hierarchy), so I'm thinking it makes sense to not include them.

Reply:
I am a bit on the fence here since this means the information users see in other places (repr for example) will be inconsistent with the information they see here. I do agree that the standard tags don't provide much additional value, however.

If we decide to remove them from the plot, I think we should document this behavior in the docstring so that users can understand that the lack of standard tags in the plot is by design and not an error.

Reply:
Yes, and in addition to inconsistencies between here and other representations, I think another worry with losing the standard tags is that they generally are how we present input and return types in primitives.

If a user is trying to look through an entityset at this high of a level and figure out what primitives they might want to use, it's on them to understand that the `'numeric'` semantic tag will apply to `Integer` and `Double`. 

But I also think that the moment we remove the logical type and semantic tag labels from this information, we're leaving the meaning up to a user's interpretation, so I'm not sure the comparison between primitive input/return types and these lines in the plot will be ideal anyway.

Reply:
My inclination is to leave off the standard tags as you have done, but include the tags that indicate index, time index and foreign key columns as I think that is important in the context of the graph. If we find that the absence of standard tags creates issues we can easily add them to the plots in a future PR.

Maybe just a quick mention in the docstring that standard tags won't be displayed in the plot?